backend/handlers/report.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+
+// Imports
+const shortid = require("shortid");
+const AccountAPI = require("../services/AccountAPI");
+const ReportAPI = require("../services/ReportAPI");
+const moment = require("moment-timezone");
+const responseUtil = require("../utilities/response");
+
+module.exports = {
+
+  // REPORT A NEW ACCOUNT //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+  NewReport: async function (event) {
+    try {
+        // if nothing was provided in the request, return a 204 HTTPS code (No content)
+        if (!event)
+            return responseUtil.Build(500, { Message: "No information was provided with the request!" });
+
+        // next parse the request
+        let request = JSON.parse(event.body);
+
+        // they need to have entered an two IDs and a message
+        if (!request.ReportedAccountID)
+          return responseUtil.Build(500, { Message: "Must include the ID of the account to be reported!" });
+
+        if (!request.AccuserID)
+          return responseUtil.Build(500, { Message: "Must include the ID of the account who is making the report!" });
+
+        if (!request.Message)
+          return responseUtil.Build(500, { Message: "Please include a message in your report!" });
```

Comment:
If we want to worry about long term maintainability, it may be simpler to make an array of strings of the required attributes, and look through to see if any is missing. Not a required change, but something to think about in case we want to continue on this project beyond the SE class